src/jobs/test-nut.yml

```diff
@@ -20,6 +21,14 @@ parameters:
     description: 'By default, the latest version of the standalone CLI will be installed. To install via npm, supply a version tag such as "latest" or "6".'
     default: ''
     type: string
+  sfdx_executable_path:
+    description: "Path to sfdx executable to be used by NUTs, defaults to ''"
+    default: ''
+    type: string
+  module_version:
+    description: "The version of the module repo to checkout, '' defaults to branch/PR"
```

Comment:
The name `module_version` and this description seem vague/confusing. Reading `The version of the module repo to checkout` makes me immediately want to go look for a repo named `module`. I'd have a hard time understanding what this was referring to without previous context. What naming this something like `repo_tag` or `repo_version`?

Reply:
@RodEsp that makes sense. I will change it to repo_tag.